tool/plugin/lib/plugin.dart

```diff
@@ -387,9 +411,9 @@ class GradleBuildCommand extends BuildCommand {
       return await super.doit();
     } finally {
       if (Platform.isWindows) {
-        return await exec('.\\gradlew.bat', ['--stop']);
+        await exec('.\\gradlew.bat', ['--stop']);
```

Comment:
wow. should we have a lint about return statements in finally clauses? @pq @munificent 
Determining which return statement should actually apply in this case is a bit confusing.

Reply:
I like that idea. I can't imagine why I typed "return" in the first place but I certainly didn't want it. Note the final statement in the protected block is a return, but you might not be able to determine that all the time.

Thanks for the review! Looks like I have a problem in the build script, which I'll fix.